types/mixpanel-browser/index.d.ts

```diff
@@ -48,6 +48,10 @@ export interface OutTrackingOptions extends ClearOptOutInOutOptions {
     delete_user: boolean;
 }
 
+export interface RegisterOptions {
+	persistent: boolean;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/mixpanel-browser/index.d.ts;
  Line: 52;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.